make/RunTests.gmk

```diff
@@ -1034,23 +1034,52 @@ define SetupRunJtregTestBody
 	$$(call LogWarn, Finished running test '$$($1_TEST)')
 	$$(call LogWarn, Test report is stored in $$(strip \
 	    $$(subst $$(TOPDIR)/, , $$($1_TEST_RESULTS_DIR))))
+
+# Read jtreg documentation to learn on the test stats categories:
+# https://github.com/openjdk/jtreg/blob/master/src/share/doc/javatest/regtest/faq.md#what-do-all-those-numbers-in-the-test-results-line-mean
+# In jtreg, "skipped:" category accounts for tests that threw jtreg.SkippedException at runtime.
+# At the same time these tests contribute to "passed:" tests.
+# In here we don't want that and so we substract number of "skipped:" from "passed:".
```

Comment:
Can you please indent this to the level of the recipe line below (using space and not tab, see (5) in https://openjdk.org/groups/build/doc/code-conventions.html).

Reply:
Fixed. Thank you.